src/lib/class.ts

```diff
@@ -104,6 +104,10 @@ export class Sections {
   locked: boolean;
   /** Currently selected section out of these. None is null. */
   selected: Section | null;
+  /** Overridden location for this particular section. */
+  roomOverride: string = "";
+
+
```

Comment:
Remove extraneous whitespace (for consistency)

Reply:
maybe we should set up prettier

Reply:
or a linter